code/game/machinery/firealarm.dm

```diff
@@ -247,6 +246,8 @@
 /obj/machinery/firealarm/Initialize(mapload, ndir = 0, building)
 	. = ..(mapload, ndir)
 
+	seclevel = "green"
```

Comment:
```seclevel = SEC_LEVEL_GREEN```
or ideally
```seclevel = get_security_level()```
are probably better options here, but I don't think the ship should start on anything other than green